### PR TITLE
General additions + Implementations

### PR DESCRIPTION
**Pull request by VocalFan**
Added Syscall 69 and 70, along with adding regkeys from 0x01010000 to 0x02420000, and adding the missing Display for `DEVENV_TOOL_USE_DEFAULT_LIB`, ending off with a use of `clippy --fix` to clean up any odd code.

Also here's your proof ultima:

image

### CODE REVIEW

File: src/kernel/src/main.rs [hunk 1]
```diff
@@ -330,20 +330,9 @@ struct Args {
     execution_engine: Option<ExecutionEngine>,
 }
 
-#[derive(Clone, ValueEnum, Deserialize)]
+#[derive(Clone, ValueEnum, Deserialize, Default)]
 enum ExecutionEngine {
+    #[default]
```
Comment (ultimaweapon) [inline src/kernel/src/main.rs:335]: `Native` cannot be used on non-x86.

File: src/kernel/src/ee/llvm/mod.rs [hunk 2]
```diff
@@ -58,18 +58,18 @@ impl ExecutionEngine for LlvmEngine {
     type SetupModuleErr = SetupModuleError;
     type GetFunctionErr = GetFunctionError;
 
-    fn set_syscalls(&self, v: Syscalls) {
+    fn set_syscalls(&self, _v: Syscalls) {
```
Comment (ultimaweapon) [inline src/kernel/src/ee/llvm/mod.rs:61]: Rename things here is not a good idea since @bayedieng might be working here.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

**Comment by VocalFan** [comment]
Bruh, Ultima, I can't believe you're already awake lel

**Comment by VocalFan** [comment]
But alright, guess clippy had some dumb moments.

File: src/kernel/src/regmgr/key.rs [hunk 3]
```diff
@@ -5,8 +5,18 @@ use std::fmt::{Display, Formatter};
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct RegKey(u32);
 
+#[allow(dead_code)] // Removes complaints about unused Regkeys.
 impl RegKey {
+    // pub const REGISTRY_VERSION: Self = Self(0);
```
Comment (ultimaweapon) [inline src/kernel/src/regmgr/key.rs:10]: Why keeping this?

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

File: src/kernel/src/regmgr/key.rs (hunk 3 above)
Comment (VocalFan) [reply src/kernel/src/regmgr/key.rs:10]: Ah crap, forgot to remove it x3 sorryyy

**Comment by VocalFan** [comment]
@ultimaweapon Ready for another check :3

**Review by ultimaweapon — APPROVED** [review]

**Pull request merged**